.gitlab-ci.yml

```diff
@@ -0,0 +1,48 @@
+---
+
+include:
+  - project: nci-gdc/gitlab-templates
+    ref: 0.2.0
+    file:
+      - templates/global/full.yaml
+      - templates/python/full.yaml
+      - templates/common/python.yaml
+
+tox:
+  stage: test
+  parallel:
+    matrix:
+      - PYTHON_VERSION: [ '3.6', '3.7', '3.8', '3.9' ]
```

Comment:
The `tox.ini` envlist include py35. Should that also be included?

Reply:
At this point, I believe we should drop py35 for every repo.